www/vitl03/sp_pilot/orders.php

```diff
@@ -0,0 +1,88 @@
+<?php require_once __DIR__ . '/class/OrdersDB.php'; ?>
+<?php require_once __DIR__ . '/class/UsersDB.php'; ?>
+<?php require_once __DIR__ . '/class/PaymentDB.php'; ?>
+<?php require_once __DIR__ . '/class/ShippingDB.php'; ?>
+<?php
+
+
+
+if (isset($_SESSION['userData']['email'])) {
+	$userEmail =  $_SESSION['userData']['email'];
+} elseif (isset($_SESSION['access_token'])) {
+	$userEmail =  $_SESSION['user_email'];
+} else {
+	$userEmail = $_SESSION['user_email'];
+}
+
+
+$ordersDB = new OrdersDB();
+$orders = $ordersDB->fetchByEmail2($userEmail);
```

Comment:
fetchAll2, fetchByEmail2, ... prejmenujte na verzi bez cislice, a to co neplati zakomentujte